pyrk/materials/fuel.py

```diff
@@ -0,0 +1,28 @@
+from ur import units
+from material import Material
+from density_model import DensityModel
+
+class Fuel(Material):
```

Comment:
My concern about this Fuel class still stands. It should have a different name. Like TRISOMatrix or something.